modules/rosa-policy-disaster-recovery.adoc

```diff
@@ -14,3 +13,75 @@ All disaster recovery requires that the customer use best practices for deployin
 One single-zone cluster will not provide disaster avoidance or recovery in the event of an availability zone or region outage. Multiple single-zone clusters with customer-maintained failover can account for outages at the zone or at the regional level.
 
 One multi-zone cluster will not provide disaster avoidance or recovery in the event of a full region outage. Multiple multi-zone clusters with customer-maintained failover can account for outages at the regional level.
+
+[cols="2a,3a,3a" ,options="header"]
+|===
+|Resource
+|Service responsibilities
+|Customer responsibilities
+
+|Virtual networking management
+|**Red Hat**
+
+- Restore or recreate affected virtual network components that are necessary for the platform to function.
+|- Configure virtual networking connections with more than one tunnel where possible for protection against outages as recommended by the public cloud provider.
+- Maintain failover DNS and load balancing if using a global load balancer with multiple clusters.
+
+|Virtual Storage management
+|**Red Hat**
+
+- For ROSA clusters created with IAM user credentials, back up all Kubernetes objects on the cluster through hourly, daily, and weekly volume snapshots.
+
+- For ROSA clusters created with IAM user credentials, back up persistent volumes on the cluster through daily and weekly volume snapshots.
+
+|- Back up customer applications and application data.
+
+|Virtual compute management
+|**Red Hat**
+
+- Monitor the cluster and replace failed Amazon EC2 control plane or infrastructure nodes.
+
+- Provide the ability for the customer to manually or automatically replace failed worker nodes.
```

Comment:
Red Hat doesn't replace failed worker nodes?

Reply:
I am going to make a separate ticket to address these additions.